spec/board_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Board do
+
+  let (:board) { Board.new }
```

Comment:
Unit test isolation: could you use doubles to isolate these tests from `Board`'s collaborator units?